pxr/usd/usd/testenv/testUsdValidationRegistryPy.py

```diff
@@ -0,0 +1,113 @@
+#!/pxrpythonsubst
+#
+# Copyright 2024 Pixar
+#
+# Licensed under the terms set forth in the LICENSE.txt file available at
+# https://openusd.org/license.
+
+import os
+import unittest
+
+from pxr import Plug, Usd
+
+
+class TestUsdValidationRegistryPy(unittest.TestCase):
+    VALIDATOR1_NAME = "testValidationPlugin:TestValidator1"
+    VALIDATOR2_NAME = "testValidationPlugin:TestValidator2"
+    VALIDATOR_SUITE_NAME = "testValidationPlugin:TestValidatorSuite"
+    PLUGIN_NAME = "testValidationPlugin"
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        Plug.Registry().RegisterPlugins(os.getcwd())
+        Usd._register_test_validators()
+
+    def test_query_validators_and_suites(self):
+        validation_registry = Usd.ValidationRegistry()
```

Comment:
The variable names also need to do follow camelCase convention. Thanks

Reply:
I'll fix them.

Reply:
Thanks @roggiezhang-nv.